mantle/kola/tests/ignition/luks.go

```diff
@@ -93,10 +93,7 @@ func setupTangMachine(c cluster.TestCluster) ut.TangServer {
 
 	// TODO: move container image to centralized namespace
 	// container source: https://github.com/mike-nguyen/tang-docker-container/
```

Comment:
Are we planning to address this part too?

Reply:
It's already done :) 
https://github.com/jbtrystram/kola-container-images/tree/main/tang